src/components/Quotes.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+
+const apiUrl = 'https://api.api-ninjas.com/v1/quotes?category=education';
+const apiKey = 'bE08nSscaJlysmr9PF2nRA==Rf2z5asHSprjI7m4';
+const Quotes = () => {
+  const [post, setPost] = useState(null);
+  const [loading, setLoading] = useState(true);
```

Comment:
- Optional.
I would recommend setting the loading state to true only when you start fetching the data. That way even if the useEffect doesn't run, it will not show the loading state forever.